app/models/user.rb

```diff
@@ -59,6 +59,7 @@ class User < ActiveRecord::Base
   has_many :group_requests, dependent: :delete_all
   has_many :muted_user_records, class_name: 'MutedUser', dependent: :delete_all
   has_many :ignored_user_records, class_name: 'IgnoredUser', dependent: :delete_all
+  has_many :invited_users, dependent: :delete_all
```

Comment:
This should be a `has_one`? A user can only be invited once.

Reply:
Actually a user can be invited multiple times via "bulk invite from file" feature where an invite is sent to grant user group access.

Reply:
Hmm how is that possible when we have a unique index on `:invite_id` and `:user_id` on the `invited_users` table.

Reply:
Aha, I think we updated the bulk invite code to grant access to groups to existing user and do not create new invite record.

I'll update the association to `has_one`.